push-api/JSONParser/README.md

```diff
@@ -1,14 +1,28 @@
-### General usage:
+# Crawl and Push
+
+Scanning a folder and push every item to the Push-API
+
+### Usage
 
 The main file is JSONParser.py
+It will execute a 4 step process:
 
-You will need to add the PyYAML lib
+1: read the config file (yamlParser.py)
+2: scan a folder and create a JSON (Reader.py) (optional if docs.json is already populated)
+3: parse the JSON file (Parser.py)
+4: loop through the array in the JSON file and push (Pusher.py) info to the Push-API
 
+### Requirement
+
+Python 3.*
+You will need to add the PyYAML lib
 Before you run the script, fill all the nodes in the config.yml
 
+### Installation
+
 * The access token can be created from the Cloud Administration Console
 * The source needs to be a PUSH-type source
-* The url node is to find the item. If you need the url to be dynamic, you will need to add your logic to the Parser.readDocument method. Please be nice and add a new class for this logic :)
-* The array is the key which should contain all the documents to be indexed
+* The array node is the key of an array in a JSON file which should contain all the documents to be indexed
```

Comment:
Put "array" in code for emphasis, or bold, and do the same for the other configurable properties too.

Either `array` or **array**

I can't understand what the phrase means though. Maybe "array" could have a better name.